sources/models/currentStory.js

```diff
@@ -4,7 +4,10 @@ var mongoose = require('mongoose');
 var currentStorySchema = new mongoose.Schema({
 	authorId: String,
 	authorName: String, 
-	authorLogo: String,
+	authorLogo: [{
+		name:String,
```

Comment:
Why is authorLogo an array? And what is the purpose of the name paramter in authorLogo?

Reply:
Logo is an array of Objects because, the entire system is configured to treat theme inputs in this structure.
name is the type of logo and logo will contain the corresponding link. (P.S. type cant be used a key as its an identifier)
I've referenced this in a different comment